x-pack/plugins/observability/public/components/shared/exploratory_view/header/add_to_case_action.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useCallback } from 'react';
+import { toMountPoint, useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
+import { ObservabilityAppServices } from '../../../../application/types';
+import { AllCasesSelectorModalProps } from '../../../../../../cases/public';
+import { TypedLensByValueInput } from '../../../../../../lens/public';
+import { useAddToCase } from '../hooks/use_add_to_case';
+import { Case, SubCase } from '../../../../../../cases/common';
+
+export interface AddToCaseProps {
+  timeRange: { from: string; to: string };
+  lensAttributes: TypedLensByValueInput['attributes'] | null;
+}
+
+export function AddToCaseAction({ lensAttributes, timeRange }: AddToCaseProps) {
+  const kServices = useKibana<ObservabilityAppServices>().services;
+
+  const { cases, http } = kServices;
+
+  const getToastText = useCallback(
+    (thaCase) => toMountPoint(<CaseToastText theCase={thaCase} basePath={http.basePath.get()} />),
```

Comment:
What does `toMountPoint` do?

Should `thaCase` be `theCase`? If not, what does tha mean?

Reply:
`toMountPoint` is a util exposed by kibana when you need work with toasts etc and you need to mount react elements.